packages/wrangler/src/api/dev.ts

```diff
@@ -228,6 +228,8 @@ export async function unstable_dev(
 	const devServer = await run(
 		{
 			FILE_BASED_REGISTRY: fileBasedRegistry,
+			// TODO: can we make this work?
```

Comment:
if TODO is relevant, should we keep track of it in a separate issue?

Reply:
https://github.com/cloudflare/workers-sdk/issues/7369